src/components/Alert/Alert.js

```diff
@@ -19,26 +22,31 @@ class Alert extends React.Component {
     dismissible: PropTypes.bool.isRequired,
     idString: PropTypes.string,
     analyticsString: PropTypes.string,
-    children: PropTypes.node.isRequired
-  };
-
-  ref = React.createRef();
+    children: PropTypes.node.isRequired,
+    handleAlertDismissed: PropTypes.func
+  }
 
   get className() {
     const { type, className } = this.props
     const baseClass = sparkComponentClassName('Alert')
     const variantClass = sparkComponentClassName('Alert', null, type)
+    const displayClass = sparkClassName('utility', 'Display', null, 'none')
 
-    return classnames(baseClass, variantClass, { [className]: className })
+    return classnames(baseClass, variantClass, { [className]: className, [displayClass]: this.state && this.state.dismissed })
```

Comment:
why would this not have a state?

Reply:
honestly I'm not sure; I was thinking that I needed to define `state` in the component, which I did, but the tests that were using `<Alert>` were still saying that `state` was null.  So as a safeguard I added this check for if state existed.  When running this component in the UI, I don't get this error.

Reply:
that's really weird. it might be worth investigating a bit cause you did define it and it should always be there

Reply:
I figured this out; because the state was defined as static, I needed to lookup state on the class itself (`Alert.state.dismissed`).  But `this.setState()` sets `state` on the class instance, so I just created a constructor to init `state` instead.